bluepyopt/ephys/locations.py

```diff
@@ -133,7 +133,7 @@ def instantiate(self, sim=None, icell=None):
     def __str__(self):
         """String representation"""
 
-        return '%s[%s]' % (self.seclist_name, self.sec_index)
+        return '%s: %s[%s]' % (self.name, self.seclist_name, self.sec_index)
```

Comment:
It's mainly adding the name that's stored by the base class, so it shouldn't change the structure that much.  Is there a particular example that will change the view?